controllers/sriovnetworknodepolicy_controller.go

```diff
@@ -106,6 +106,12 @@ func (r *SriovNetworkNodePolicyReconciler) Reconcile(ctx context.Context, req ct
 		return reconcile.Result{}, err
 	}
 
+	// Fetch the default SriovOperatorConfig
+	defaultOpConf := &sriovnetworkv1.SriovOperatorConfig{}
+	if err := r.Get(ctx, types.NamespacedName{Namespace: vars.Namespace, Name: constants.DefaultConfigName}, defaultOpConf); err != nil {
+		return reconcile.Result{}, err
```

Comment:
nit: Maybe we can log here something like "no operator config found, waiting for creation" and explicitly requeue with some delay, e.g. 5 seconds?

Reply:
i dont think we need to requeue, we will be called as soon as the object is created.

im thinking just to add log and return a successful reconcile WDYT ?

Reply:
We are inside policy controller. This Get call is for OperatorConfig object. Maybe I'm missing something, but I don't see that policy controller watches for OperatorConfig object.

Reply:
@ykulazhenkov the scenario you're referring to is:
a. The user deploys the operator
b. the user creates an SriovNetworkNodePolicy
c. The user creates the SriovOperatorConfig

right?

In this case, the policy would not get reconciled until another SriovNetworkNodePolicy is created/modified, I think.

Reply:
@zeeke 

Yes, I'm referring to this case. 
My concern here is that if the SriovNetworkNodePolicy controller fails multiple times in a row because there is no Operator config, then retry with delay will be used. If there were many failures, then the retry interval would be quite large. I propose explicitly sending a message to the queue again with a non-error exit to ensure we will retry reconciliation quite soon. This means that we will do a periodic polling in the  SriovNetworkNodePolicy controller until operator's config is created. Current implementation will work just fine, but in some edge-cases it may introduce unneeded delay.

Reply:
>We are inside policy controller. This Get call is for OperatorConfig object. Maybe I'm missing something, but I don't see that policy controller watches for OperatorConfig object.

yes, you are right, i thought you are referring to the operator config controller  :\

Reply:
Added a check here. requeue after 5 seconds